framework/src/controller/helpers/configurator/configurator.js

```diff
@@ -0,0 +1,148 @@
+const _ = require('lodash');
```

Comment:
prefer not to use `lodash` but for this PR it's fine =p

Reply:
Ok. But we should make a general rule, if something not available natively, then we must use `lodash`, its still much better than custom implementation.

Reply:
yea if it's real functionality (like shuffle or difference) it's fine i think, but we should import it separately case by case, but we can handle it on next version 👍